README.md

```diff
@@ -63,6 +63,10 @@ In both cases if you are in doubt, please disable/remove the `INSIGHTS_CLIENT_ID
 
 ## Integrations
 
+### Stage usage
+
+Start `insights-mcp` with `--stage` and set `INSIGHTS_STAGE_PROXY_URL` accordingly
```

Comment:
should there be a `--stage` option connecting to private deployment in the public repo? Wouldn't it be better if someone wants to use non-prod deployment of insights (local, stage, ephemeral, fedramp, etc.) just override INSIGHTS_BASE_URL and INSIGHTS_TOKEN_ENDPOINT env vars? Treating stage differently is not the best approach in my opionion, as well as exposing internal urls in codebase.